packages/manager/modules/anthos/src/dashboard/host/host-table/host.class.js

```diff
@@ -0,0 +1,44 @@
+import { STATUS } from './constants';
+
+export default class Host {
+  constructor({
+    id,
+    name,
+    profile,
+    privateIp,
+    rack,
+    stateful,
+    additional,
+    status,
+    serviceName,
+  }) {
+    Object.assign(this, {
+      id,
+      name,
+      profile,
+      privateIp,
+      rack,
+      stateful,
+      additional,
+      status,
+      serviceName,
+    });
+  }
+
+  get statusGroup() {
+    switch (this.status) {
+      case 'BAREMETAL_STATUS_UNSPECIFIED':
```

Comment:
may you should create a enum and move them to a constant file

Reply:
This is done. Thanks